src/instructlab/config/init.py

```diff
@@ -222,7 +222,7 @@ def hw_auto_detect() -> tuple[str | None, int | None, _train, bool]:
     edited_cfg = False
     # try nvidia
     if torch.cuda.is_available() and torch.version.hip is None:
-        click.echo("\nDetecting Hardware...")
+        click.echo("Detecting Hardware...")
```

Comment:
nit: I know you aren't adding it here, but hardware should be lowercas to be consistent.